File: xamarin/common/Presenter/Events/LocationSelectedEventArgs.cs
using System;
using PropertyCross.Model;

namespace PropertyCross.Presenter
{
  public class LocationSelectedEventArgs : EventArgs
  {
    public LocationSelectedEventArgs (Location location)
    {
      Location = location;
    }

    public Location Location { private set; get; }
  }

  public class RecentSearchSelectedEventArgs : EventArgs
  {
    public RecentSearchSelectedEventArgs (RecentSearch recentSearch)
    {
      RecentSearch = recentSearch;
    }

    public RecentSearch RecentSearch { get; private set; }
  }
}

File: native/android/PropertyCross/app/src/main/java/com/propertycross/android/events/LocationSelectedEventArgs.java
package com.propertycross.android.events;

import com.propertycross.android.model.Location;

public class LocationSelectedEventArgs {
  private Location location;

  public LocationSelectedEventArgs(Location location) {
    this.location = location;
  }

  public Location getLocation() {
    return location;
  }
}
